ycms/cms/models/ward.py

```diff
@@ -52,6 +52,16 @@ def available_beds(self):
         """
         return sum(room.available_beds for room in self.rooms.all())
 
+    @cached_property
+    def occupied_beds(self):
+        """
+        Helper property for accessing the wards occupied bed count
+
+        :return: number of occupied beds in the ward
+        :rtype: int
+        """
+        return self.total_beds() - self.available_beds()
```

Comment:
```suggestion
        return self.total_beds - self.available_beds
```
Feel free to ignore this - both versions work, without `()` is more common, but it's mostly down to preferrence, so either is fine!